javaserver/inserts.sql

```diff
@@ -0,0 +1,3 @@
+INSERT INTO facilities VALUES ('Tvättstuga');
+INSERT INTO facilities VALUES ('Gym');
+INSERT INTO facilities VALUES ('Lokaler');
```

Comment:
Vill jag helst ha alla users som inserts också, jag kan fixa det om du vil @0x3D

Reply:
Jag är efterbliven, såg dem inte tools mappen!

Reply:
Added a setup.sql so it is easier to setup database!